frontend-service-client/src/main/java/com/amazonaws/blox/model/transform/DescribeEnvironmentRequestProtocolMarshaller.java

```diff
@@ -22,7 +22,7 @@
 @SdkInternalApi
 public class DescribeEnvironmentRequestProtocolMarshaller implements Marshaller<Request<DescribeEnvironmentRequest>, DescribeEnvironmentRequest> {
 
-    private static final OperationInfo SDK_OPERATION_BINDING = OperationInfo.builder().protocol(Protocol.API_GATEWAY).requestUri("/Beta/environments/{name}")
+    private static final OperationInfo SDK_OPERATION_BINDING = OperationInfo.builder().protocol(Protocol.API_GATEWAY).requestUri("/alpha/environments/{name}")
```

Comment:
This is an autogenerated change; I changed the stage name to `alpha`, to match the profile name. This brings up the concern that the stage name is hardcoded in the client, I'll reach out to the SDK team to see if we can make this configurable instead.